src/components/StateFooter/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Button, Grid, makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+    maxWidth: 1000
+  },
+  button: {
+    textTransform: "none"
+  }
+}));
+
+const StateFooter = () => {
+  const classes = useStyles();
+
+  const handleViewState = (event: any) => {};
+
+  const handleResetState = (event: any) => {};
+  
+  return (
+      <Grid
+        container
+        className={classes.root}
+        justify="space-between"
+        alignItems="flex-end"
+      >
+        <Grid item>
+          <Button className={classes.button} onClick={handleViewState}>
+            View Current State
+          </Button>
+        </Grid>
+        <Grid item>
+          <Button className={classes.button} onClick={handleResetState}>
+            Reset State
+          </Button>
+        </Grid>
+    </Grid>
```

Comment:
A grid is not necessary for one row. Rather use the Box component with a display of flex and justifyContent  of space-between. It's much simpler. Always use Box first before using other layout components.

Reply:
If you are interested: 

- [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
- https://material-ui.com/system/flexbox/